rust_src/src/threads.rs

```diff
@@ -24,6 +25,14 @@ impl ThreadStateRef {
     pub fn name(self) -> LispObject {
         LispObject::from(self.name)
     }
+
+    #[inline]
+    pub fn alive_p(self) -> LispObject {
+        match self.m_specpdl == ptr::null_mut() {
+            true => LispObject::constant_nil(),
+            false => LispObject::constant_t(),
+        }
```

Comment:
You could also just do an if/else, but this will work just fine.

Reply:
Yeah, the if/else is definitely more idiomatic. On it.